stdlib/public/core/ArrayCast.swift

```diff
@@ -170,7 +174,7 @@ ElementwiseBridging:
   while false
   
   // Don't destroy anything we never created.
-  buf.count = p - buf.firstElementAddress
+  buf.count = p - buf.firstElementAddress!
```

Comment:
Actually, never mind.  `subscriptBaseAddress` is adjusted for non-zero based indices of slices.